modules/policy-criteria.adoc

```diff
@@ -29,524 +30,831 @@ In this table:
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | CVE
 | Common Vulnerabilities and Exposures, use it with specific CVE numbers.
-| All
-| ✓
-| ✓
-| ✓
-| Build
+| CVE
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
 
 | Image Component
 | Name and version number of a specific software component present in an image.
-| All
-| ✓
-| ✕
-| ✓
-| Build
+| Image Component
+| key=value +
+
+Value is optional. +
 
+If value is missing, it must be in format "key=".
+| Regex, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Image OS
+| Name and version number of the base operating system of the image. For example, `alpine:3.17.3``
 | Image OS
-| Name and version number of the base operating system of the image.
-| 3.0.47 and newer
-| ✓
-| ✓
-| ✓
-| Build
+| String
+| Regex, +
+NOT, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Require image label
+| Ensure the presence of a Docker image label. The policy triggers if any image in the deployment does not have the specified label. You can use regular expressions for both key and value fields to match labels. The 'Require Image Label' policy criteria only works when you integrate with a Docker registry. For details about Docker labels see Docker documentation, https://docs.docker.com/config/labels-custom-metadata/.
+| Required Image Label
+| key=value +
+
+Value is optional. +
+
+If value is missing, it must be in format "key=".
+| Regex, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+| Disallow image label
+| Ensure that a particular Docker image label is NOT used. The policy triggers if any image in the deployment has the specified label. You can use regular expressions for both key and value fields to match labels. The 'Disallow Image Label policy' criteria only works when you integrate with a Docker registry. For details about Docker labels see Docker documentation, https://docs.docker.com/config/labels-custom-metadata/.
+| Disallowed Image Label
+| key=value +
+
+Value is optional. +
+
+If value is missing, it must be in format "key=".
+| Regex, +
+AND, OR
+| *Build*, +
+*Deploy*, +
+*Runtime* (when used with a Runtime criterion)
+
+6+| *Section: Container configuration*
 
 | Environment Variable
 | Check environment variables by name or value.
-| All
-| ! only for key and value
+| Environment Variable
+| RAW=key=value to match an environment variable as directly specified in the deployment configuration with a specific key and value. `value`` can be omitted to match on only the key. +
+
+If the environment variable is not directly defined in the configuration, then the format SOURCE=KEY can be used, where SOURCE is one of SECRET_KEY, CONFIG_MAP_KEY, FIELD or RESOURCE_FIELD. In this case, criteria can only match the key and not the value.
+| ! Regex only for key and value (if using RAW) /+
```

Comment:
Is the way this is displayed with symbols correct?

Reply:
What is the `/+` supposed to mean?

Reply:
It was a typo; I have removed it in the commit I'm about to push.